src/pages/waiting-room/components/cbt-number/__tests__/cbt-number.spec.ts

```diff
@@ -0,0 +1,56 @@
+
+import { ComponentFixture, TestBed, async } from '@angular/core/testing';
+import { IonicModule } from 'ionic-angular';
+import { AppModule } from '../../../../../app/app.module';
+import { FormGroup } from '@angular/forms';
+import { CBTNumberComponent } from '../cbt-number';
+import { configureTestSuite } from 'ng-bullet';
+
+describe('CBTNumberComponent', () => {
+  let fixture: ComponentFixture<CBTNumberComponent>;
+  let component: CBTNumberComponent;
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        CBTNumberComponent,
+      ],
+      imports: [
+        IonicModule,
+        AppModule,
+      ],
+    });
+  });
+
+  beforeEach(async(() => {
+    fixture = TestBed.createComponent(CBTNumberComponent);
+    component = fixture.componentInstance;
+    component.formGroup = new FormGroup({});
+  }));
+
+  describe('Class', () => {
+    describe('cbtNumberChanged', () => {
+      it('It should emit a cbt number if the characters only contain numbers', () => {
+        spyOn(component.cbtNumberChange, 'emit');
+        const cbtNumber = '12345678';
+        component.cbtNumberChanged(cbtNumber);
+        expect(component.cbtNumberChange.emit).toHaveBeenCalledWith(cbtNumber);
+      });
+    });
+
+    describe('invalid', () => {
```

Comment:
this `describe` message doesn't make much sense.

Reply:
Wait it does. It's just testing the `invalid()` function in the `ts` file, yeah?

Reply:
yea :)